src/pykiso/lib/connectors/cc_proxy.py

```diff
@@ -85,3 +85,23 @@ def _cc_receive(self, timeout: float = 0.1, raw: bool = False) -> ProxyReturn:
             return raw_msg, source
         except queue.Empty:
             return None, None
+
+    def cc_share(self, *args: tuple, **kwargs: dict) -> None:
+        """Populate the connector's queue out with other auxiliaries
+        requests. Before populating the queue, just adapt the return
+        values to fit with current connectors return type (message,
+        source pair).
+
+        This method is currently used by the proxy auxiliary to dispatch
+        other commands coming from ohter auxiliaries.
+
+        .. note:: for the moment only handle the special case of
+            remote_id parameter, ohterwise extend this function to
+            handle more specific cases
+
+        :param args: postional arguments
+        :param kwargs: named arguments
+        """
+        message = kwargs.get("msg")
+        source = kwargs.get("remote_id")
```

Comment:
We create here a dependency to "remote_id" on proxy level. It should be done on cc_can instead

Reply:
The request (containing the remote ID) is dispatched to the other cc_proxy_connectors.
If I remove the handling of the remote ID, your other auxes will never received this message.
The disptach of th reauest to ohter auxes was done to build more complex simulation based on ohter auxes behavior.
By removing it you will lost this features.

Reply:
It should be done on cc_can instead  : It's done see cc_pcan_can.